manifests/init.pp

```diff
@@ -140,6 +140,18 @@
                 warning('Packages for Ruby 2.1 are not available from default repositories.')
               }
             }
+            /^2\.2.*$/:{
```

Comment:
Looks like. Cause the current regex would also match 2.20.0, which is the not what you want.